domains/authorization/Authorization.API/API.IntegrationTests/API.IntegrationTests/Migrations/AddTermsMigrationTests.cs

```diff
@@ -32,14 +34,37 @@ public async Task AddTerms_Migration_SetsTermsAcceptedToFalseForExistingOrganiza
         var applyMigration = () => migrator.MigrateAsync("20240711101829_AddTermsWithOutbox");
         await applyMigration.Should().NotThrowAsync();
 
-        var organizations = await dbContext.Organizations.ToListAsync();
-        organizations.Should().NotBeEmpty();
-        organizations.Should().AllSatisfy(org =>
+        var connectionString = dbContext.Database.GetConnectionString();
```

Comment:
https://learn.microsoft.com/en-us/ef/core/querying/sql-queries?tabs=sqlserver

There is ways to query raw sql directly on EF core if we want. So we don't have to get connection string out and make raw NpSQL connections.

Reply:
(Can't remember how we test this other places.)